src/main/webapp/style.css

```diff
@@ -1,7 +1,59 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+  margin-top: 30px;
+  width: 500px;
+}
+
+.card-container {
+  margin-right: 30px;
+  margin-left: 30px;
+  flex-shrink: 0;
+  flex-basis: 500px;
+}
+
+.card .card-description {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 32px;
+}
+
+.card .card-title {
+  font-size: 20px;
+  font-weight: 300;
+  cursor: pointer;
+  line-height: 32px;
+  display: flex;
+}
+
+.card .card-content {
+  padding: 24px;
+  border-radius: 0 0 2px 2px;
+  width: 400px;
+  display: flex;
```

Comment:
It's overkill to make this a flexbox - you just want blocks of text appearing one below the other, which is the default behavior of block elements like <p>. (Incidentally, you overrode the display: block behavior of <p> to inline...)

But making it a flexbox is causing a few weird bugs. For example, if you hover your mouse to the right of the link, you'll notice that the empty space is clickable. This is because the flexbox is making the link extend past the text.